package.json

```diff
@@ -41,6 +41,7 @@
     "postgres-migrations": "^5.3.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.10.1",
```

Comment:
MUI includes an icon package, which we already have imported. Check it out and see if they have compatible icons for you to use instead.

Reply:
got it!